bootstrap.py

```diff
@@ -7,6 +7,7 @@
 Use `awk -F'# >' 'NF>1 {print $2}' ./bootstrap.py` to see the flow.
 """
 
+import yum as yumApi
```

Comment:
I was debating on which I hated more:

- renaming our `yum` function.  (Which means a LOT of code gets touched)
- renaming the import. 

They both suck.